src/index.js

```diff
@@ -1,28 +1,52 @@
 import './index.css';
+import {
+  createGameScores,
+  getGameScores,
+  saveGameScore,
+} from '../modules/gameFunc.js';
 
-const submit = document.querySelector('.submit');
-const nameInput = document.getElementById('name');
-const scoreInput = document.getElementById('score');
-const scoresList = document.getElementById('scores');
+// 1. Create a new game and save its ID
+// 2. Add functionalities for refreshing and submitting the game
 
-// create a new score item for the list
-const createScores = (name, score) => {
-  const li = document.createElement('li');
-  li.textContent = `${name}: ${score}`;
-  return li;
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+let gameId;
+const createGame = async () => {
+  const response = await fetch(`${apiUrl}games/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ name: 'My Game' }),
+  });
+  const data = await response.json();
+  gameId = data.result.slice(-5);
```

Comment:
**OPTIONAL** - You seem to be creating the wrong game ID due to the slice method used here. I suggest you properly extract the game ID gotten from the API. Please fix this 🙏 
**OPTIONAL** - Also, since you are trying to create a unique game ID for every new user dynamically, I suggest storing the generated game ID in the local storage and using it when necessary, so the user does not always get a new game ID when he refreshes his browser. Please fix this 😄